src/app/binding/BindingTable.cpp

```diff
@@ -0,0 +1,237 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "app/binding/BindingTable.h"
+
+#include <app-common/zap-generated/attribute-type.h>
+#include <app/server/Server.h>
+#include <lib/support/logging/CHIPLogging.h>
+
+namespace {
```

Comment:
Why not put this anonymous namespace inside the chip::app namespace below and lose the extra prefixing?